tests/framework/prototype.py

```diff
@@ -192,6 +196,25 @@ def assert_rule(self, custom_resource=None, cr_namespace='default'):
             " python -c \"import ray; ray.init(); print(ray.cluster_resources())\"")
         assert rtn == 0
 
+class CurlServiceRule(Rule):
+    """"Using curl to access the deployed application on Ray service"""
+    def assert_rule(self, custom_resource=None, cr_namespace='default'):
+        # Create a pod for running curl command, because the service is not exposed.
+        os.system("kubectl run curl --image=radial/busyboxplus:curl --command --\
+           /bin/sh -c \"while true; do sleep 10;done\"")
+        while True:
+            resp = client.CoreV1Api().read_namespaced_pod(name="curl",
+                                                    namespace='default')
+            if resp.status.phase != 'Pending':
+                break
+            time.sleep(1)
+        output = subprocess.check_output("kubectl exec curl -- \
+            curl -X POST -H 'Content-Type: application/json' \
+            rayservice-sample-serve-svc.default.svc.cluster.local:8000 \
```

Comment:
Do not hard-code the service's name.

https://github.com/ray-project/kuberay/blob/0bd28e758fe89c69ab3e207baa09644562a393c6/ray-operator/controllers/ray/utils/util.go#L135-L137